core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -2213,16 +2239,21 @@ object Parser {
       }
     }
 
+    /** capture parser p repeatedly, at least min times, at most max times
+      */
     final def repCapture[A, B](
         p: Parser[A],
         min: Int,
+        max: Int,
         state: State,
         append: Appender[A, B]
     ): Boolean = {
+      //require(max >= 0)
```

Comment:
Is it valid for `max < min`? If so, what do we return? It seems like it has to be `>= min` or if it is really `max - 1` then it has to be `min  < max`

Reply:
Let's remove the commented code. If we are going to put assertions let's put them before parse time (at parser creation time).